transport/http/outbound.go

```diff
@@ -147,6 +147,11 @@ func (o *Outbound) Transports() []transport.Transport {
 	return []transport.Transport{o.transport}
 }
 
+// Chooser returns the outbound's peer chooser.
+func (o *Outbound) Chooser() peer.Chooser {
```

Comment:
I’ll examine the possibility of making a yarpctest utility for vending out config tests.